test/model/course_test.dart

```diff
@@ -191,6 +213,39 @@ void main(){
     });
   });
 
+  test('12', () {
```

Comment:
There is a ')' missing at every test...

Reply:
That's the reason why travis is failing. So just close all the opend brackets and travis should be fixed.